server/sonar-server/src/main/java/org/sonar/server/db/migrations/PlatformDatabaseMigration.java

```diff
@@ -64,15 +68,34 @@ public PlatformDatabaseMigration(RubyBridge rubyBridge, PlatformDatabaseMigratio
 
   @Override
   public void startIt() {
```

Comment:
the condition at the beginning of `startIt` is a fail fast which does not imply any concurrency management.
method `PlatformDatabaseMigration#tryLock()` is actually useless since the only difference is the respect of the fairness policy which we do not use anyway in this class. I'll remove it.